### PR TITLE
FHE 2026 and post updates, sync-up

### PR DESCRIPTION
**Pull request by jr1221**
## Changes

_Explanation of changes goes here_

## Notes

_Any other notes go here_

## Test Cases

- Case A
- Edge case
- ...

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please reach out to your Project Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)

### CODE REVIEW

File: odysseus_tree/linux_patches/nrc_halow/01-yeet_dma_align_check.patch
Comment (caiodasilva2005) [inline odysseus_tree/linux_patches/nrc_halow/01-yeet_dma_align_check.patch:1]: was a fix found for this or was this just unused?

File: odysseus_tree/overlays/rootfs_overlay_ap/etc/hostapd.conf [hunk 1]
```diff
@@ -4,8 +4,8 @@ interface=wlan0
 ssid=NER_Halow
 hw_mode=g
 #
-# wds_sta=1
-bridge=br0
+#wds_sta=1
+#bridge=br0
```
Comment (caiodasilva2005) [inline odysseus_tree/overlays/rootfs_overlay_ap/etc/hostapd.conf:8]: why is bridge removed here? Is this cause separation of IPs with HaLow and 2.4 GHz?

File: odysseus_tree/overlays/rootfs_overlay_tpu/etc/udev/rules.d/49-usb-ip.rules [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # usbipd must be running AND and usbip_host must be manually modprobed
-#SYMLINK+="ttyCerberus",SUBSYSTEMS=="usb",MODE="0666",KERNEL=="1-1.3",RUN{program}+="/usr/sbin/usbip bind -b %k"
+SYMLINK+="ttyCerberus",SUBSYSTEMS=="usb",MODE="0666",KERNEL=="1-1.2",RUN{program}+="/usr/sbin/usbip bind -b %k"
```
Comment (caiodasilva2005) [inline odysseus_tree/overlays/rootfs_overlay_tpu/etc/udev/rules.d/49-usb-ip.rules:2]: is this just a better way to set up the udev rule?

File: odysseus_tree/overlays/rootfs_overlay_tpu/sbin/quicknet [hunk 3]
```diff
@@ -1,75 +1,67 @@
...
+        -p /var/run/wpa_supplicant.wlan1.pid \
+        -- -i wlan1 -c /etc/wpa_supplicant_base.conf -f /var/log/wpa1.log
+
+    # wlan0 is 900 MHz
+    ip link set dev wlan0 up
+    start-stop-daemon -S -x /usr/sbin/wpa_supplicant -b -m \
+        -p /var/run/wpa_supplicant.wlan0.pid \
+        -- -i wlan0 -c /etc/wpa_supplicant_halow.conf -f /var/log/wpa0.log
+
+    # LTE
```
Comment (caiodasilva2005) [inline odysseus_tree/overlays/rootfs_overlay_tpu/sbin/quicknet]: oh boy. LTE is here. QMI looks pretty lit

File: odysseus_tree/overlays/rootfs_overlay_ap/etc/hostapd.conf (hunk 1 above)
Comment (jr1221) [reply odysseus_tree/overlays/rootfs_overlay_ap/etc/hostapd.conf:8]: Ya the bridge was causing issues.  I want to make a full confluence page for the new network, will do that soon.

File: odysseus_tree/overlays/rootfs_overlay_tpu/etc/udev/rules.d/49-usb-ip.rules (hunk 2 above)
Comment (jr1221) [reply odysseus_tree/overlays/rootfs_overlay_tpu/etc/udev/rules.d/49-usb-ip.rules:2]: The index 1-1.2 identifies the USB by the port its in.  Didnt really need to comment it out.

File: odysseus_tree/linux_patches/nrc_halow/01-yeet_dma_align_check.patch
Comment (jr1221) [reply odysseus_tree/linux_patches/nrc_halow/01-yeet_dma_align_check.patch:1]: The file this was in was heavily refactored by linux people, so the patch no longer applies.  Also this patch was originally found from a bug report so it was always needing to get fixed.
